src/server.js

```diff
@@ -48,6 +49,25 @@ module.exports = function() {
 			maxAge: 86400 * 1000,
 		}));
 
+	app.get("/uploads/:name/:slug*?", (req, res) => {
+		const name = req.params.name;
```

Comment:
Add a length (and perhaps a-z0-9?) check to match the hash format we generate. This will avoid trying to request `.tmp` folder as well.

Reply:
Im a little hesitant to enforce too strict rules regarding this because plugins may want to do something differently and there’s no easy way for them to interface with express. But that’s a ways off, so I’m fine with it for now

Reply:
That's one of the reasons I wanted you to move this route to the upload plugin, because I expect other plugins to provide their own route if they need it. I expect other plugins to provide custom URLs though (imgur, etc).

Reply:
Is `[0-9a-fA-F]+` fine with you?

Need to figure out best way to access express in `uploader.js` in order to handle routing there

Reply:
Fixed with what we discussed on IRC

Reply:
For the route, you can do `Uploader.router(app);`